components/Nav/Nav.test.tsx

```diff
@@ -64,13 +77,37 @@ describe('Nav', () => {
 
     const title = screen.getByTestId('title');
     const logo = screen.getByTestId('logo-nav');
+    const preferencesLink = screen.getByTestId('settings-link');
     const signOutButton = screen.getByTestId('sign-out-button');
 
+    expect(preferencesLink).toBeInTheDocument();
```

Comment:
Could combine line 80 with this line for cleaner code

Reply:
Could do that for all of the `expects` but is it better to have them separate for legibility or condensed code? 🤔

Reply:
@ryandotfurrer I like it cause of legibility.  It is starting to stand out to me as a preference rather than really clean code.  Thoughts @choir27?

Reply:
✅ I did change it to see what it would look like:

```
  expect(screen.getByTestId('settings-link')).toBeInTheDocument();
  expect(screen.getByTestId('title')).toBeInTheDocument();
  expect(screen.getByTestId('logo-nav')).toBeInTheDocument();
  expect(screen.getByTestId('sign-out-button')).toBeInTheDocument();
    ```